Rakefile

```diff
@@ -72,7 +70,7 @@ namespace :inspec do
     puts stdout
 
     %w{errors}.each do |type|
-      abort("InSpec check failed with syntax #{type}!") if !!(/[1-9]\d* #{type}/ =~ stdout)
+      abort("InSpec check failed with syntax #{type}!") if /[1-9]\d* #{type}/ =~ stdout
```

Comment:
`!!` was added to explicitly turn the results from `=~` into a boolean. With this change you are relying on the truthiness of the return value which would either be `0` or a `#<MatchData...>`.